discover schema test broken after new metadata addition.

```
======================================================================
FAIL: test_discoverSchema (testDiscoverSchema.DiscoverSchemaTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/TresAmigosSD/SMV/src/test/python/testDiscoverSchema.py", line 38, in test_discoverSchema
    self.assertEqual(res, exp)
AssertionError: '@del[54 chars]teger @metadata={"smvDesc":"1"}\nb: String @me[55 chars]"a"}' != '@del[54 chars]teger\nb: String\nc: String'
  @delimiter = ,
  @has-header = true
  @quote-char = "
- a: Integer @metadata={"smvDesc":"1"}
- b: String @metadata={"smvDesc":"2"}
- c: String @metadata={"smvDesc":"a"}+ a: Integer
+ b: String
+ c: String
```

--- a/src/test/python/testDiscoverSchema.py
+++ b/src/test/python/testDiscoverSchema.py
@@ -31,8 +31,8 @@ class DiscoverSchemaTest(SmvBaseTest):
         exp = ('@delimiter = ,\n'
                '@has-header = true\n'
                '@quote-char = "\n'
-               'a: Integer\n'
-               'b: String\n'
-               'c: String')
+               'a: Integer @metadata={"smvDesc":"1"}\n'
+               'b: String @metadata={"smvDesc":"2"}\n'
+               'c: String @metadata={"smvDesc":"a"}')
 
         self.assertEqual(res, exp)

